src/components/GuardianCard.story.tsx

```diff
@@ -0,0 +1,16 @@
+import { Story, Meta } from '@storybook/react';
+import { GuardianCard as GuardianCardComponent } from './GuardianCard';
+import { ShieldPurple, Gray3, TextLight } from '../colors';
+
+export default {
+    title: 'App/GuardianCard',
+    component: GuardianCardComponent,
+} as Meta;
+
+export const GuardianCard: Story = () => {
+    return <GuardianCardComponent 
+        color = {ShieldPurple}
+        address = '0xca...4d67'
```

Comment:
similar comment here -> would pass in a full address & within the component use the `shortenAddress` function